crates/ruma-identifiers/src/client_secret.rs

```diff
@@ -0,0 +1,172 @@
+//! Client secret identifier.
+
+use std::{convert::TryFrom, fmt, mem, rc::Rc, str::FromStr, sync::Arc};
+
+use ruma_identifiers_validation::client_secret::validate;
+
+/// A client secret.
+///
+/// Session IDs in Matrix are opaque character sequences of `[0-9a-zA-Z.=_-]. Their length must
```

Comment:
Unclosed inline code block.

```suggestion
/// Session IDs in Matrix are opaque character sequences of `[0-9a-zA-Z.=_-]`. Their length must
```